tasks/add-fbc-contribution/README.md

```diff
@@ -14,6 +14,9 @@ Task to create a internalrequest to add fbc contributions to index images
 | targetIndex    | targetIndex value updated by update-ocp-tag task                                          | No       | -                    |
 | resultsDirPath | Path to results directory in the data workspace                                           | No       | -                    |
 
+## Changes in 3.2.1
+* fixing checkton/shellcheck linting issues in the task and test
```

Comment:
I wanted to say that you should capitalize Fixing. But then I noticed that we haven't really been consistent in this file. Would you mind capitalizing all of them? If not, it's ok. Just the first letter after each `* `

Reply:
Certainly, on it.

Reply:
Done!

Reply:
Thanks!